publisher-service/src/main/resources/application.conf

```diff
@@ -29,6 +29,8 @@ publisher.service {
 		templates = "nl.idgis.publisher.database.ExtendedPostgresTemplates"
 		user = ${PG_USER}
 		password = ${PG_PASSWORD}
+		setApplicationName = true
+		setApplicationName = ${?SET_APPLICATION_NAME}

```

Comment:
Suggestie: toevoegen van PG_ prefix toevoegen aan de environment naam zodat het duidelijk(er) is dat er sprake is van een database parameter.